web_ui/packages/smart-tools/src/utils/wasm-utils.ts

```diff
@@ -4,27 +4,19 @@
 export interface SessionParameters {
     numThreads: number;
     executionProviders: string[];
-    wasmRoot?: string | Record<string, string>;
+    wasmRoot?: Promise<string | Record<string, string>>;
 }
 
-const wasmPaths = {
-    'ort-wasm.wasm': new URL('../../../../node_modules/onnxruntime-web/dist/ort-wasm.wasm', import.meta.url).toString(),
-    'ort-wasm-simd.wasm': new URL(
-        '../../../../node_modules/onnxruntime-web/dist/ort-wasm-simd.wasm',
-        import.meta.url
-    ).toString(),
-    'ort-wasm-threaded.wasm': new URL(
-        '../../../../node_modules/onnxruntime-web/dist/ort-wasm-threaded.wasm',
-        import.meta.url
-    ).toString(),
-    'ort-wasm-simd-threaded.wasm': new URL(
-        '../../../../node_modules/onnxruntime-web/dist/ort-wasm-simd-threaded.wasm',
-        import.meta.url
-    ).toString(),
+const getWasmPaths = async () => {
```

Comment:
this change was necessary because if we don't lazy load the paths, then the URL executes immediately when the file is loaded, which causes a lot of issues when we're on the jest context. To circumvent that, it was recommended to move the calls to URL to a different file, and lazy load it to avoid jest executing this. Actual error: "import.meta cannot be executed outside of a module" .

Before this i also tried using `moduleNameWrapper` but then i would need to mock ALL utils one by one, which broke hundreds of tests.

Reply:
I don't fully understand why this is necessary now (and not before?) or why we only need to do this when getting the wasm paths and not when getting the paths for the ritm or sam models.
Can you elaborate on that?

Reply:
Because now the utils are on the package, along with unit tests, which brings jest into the mix. Not only geti now imports these utils, but we also test these utils on the smart-tools package. And when jest starts to load smart-tools, it also loads this wasm-utils.ts file, which brings all sorts of issues. Before we didnt have any tests on the package, nor did we import "session" or wasm stuff from our unit tests